src/test/java/uk/gov/digital/ho/pttg/HmrcResourceTest.java

```diff
@@ -82,7 +82,7 @@ public void shouldHandleOptionalToDate() {
     public void shouldOnlyLogRedactedNino() {
         // given
         final String redactedNino = "QQ1***56C";
```

Comment:
As this format is not used anymore probably best to change it to avoid confusion - I don't think it matters if you use the new format or just called it "RedactedNino" like you did previously.